lib/Client.js

```diff
@@ -2332,15 +2332,41 @@ class Client extends EventEmitter {
     /**
     * Get the ban list of a guild
     * @arg {String} guildID The ID of the guild
-    * @returns {Promise<Array<Object>>} Resolves with an array of {reason: String, user: User}
-    */
-    getGuildBans(guildID) {
-        return this.requestHandler.request("GET", Endpoints.GUILD_BANS(guildID), true).then((bans) => {
-            bans.forEach((ban) => {
-                ban.user = new User(ban.user, this);
-            });
-            return bans;
+    * @arg {Object} [options] Options for the request
+    * @arg {String} [options.after] Only get users after given user ID
+    * @arg {String} [options.before] Only get users before given user ID
+    * @arg {Number} [options.limit] The maximum number of users to return
```

Comment:
Add the default limit here

Reply:
What's the default limit?

Reply:
https://discord.com/developers/docs/resources/guild#get-guild-bans

1000